styles/style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+
+* {
+    font-family: 'Roboto', sans-serif;
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+h1 {
+    display: flex;
+    justify-content: center;
+    width: 306px;
+    height: 47px;
```

Comment:
don't use fix height, better to add padding between title and line

Reply:
removed height and added padding